libcodechecker/libclient/product_helper.py

```diff
@@ -19,10 +19,11 @@
 
 
 class ThriftProductHelper(object):
-    def __init__(self, host, port, uri, session_token=None):
+    def __init__(self, protocol, host, port, uri, session_token=None):
         self.__host = host
         self.__port = port
-        self.transport = THttpClient.THttpClient(self.__host, self.__port, uri)
+        url = protocol+"://"+host+":"+str(port)+uri
```

Comment:
This url creating code is there multiple times, maybe worth to create a function?

Reply:
Same comment about the name of `protocol` as above.